docs/arch/adr-008-weave-to-flannel-migration.md

```diff
@@ -0,0 +1,44 @@
+# ADR 8: Weave to Flannel Migration
+
```

Comment:
Hi @laverya,

I think it is just missing add/doc the answer of the question and then it seems good enough to get merged.

Reply:
What do you mean?